src/htr2hpc/api_client.py

```diff
@@ -0,0 +1,329 @@
+import datetime
+import logging
+from collections import namedtuple
+from dataclasses import dataclass
+from time import sleep
+import pathlib
```

Comment:
Perhaps only import `Path`
```suggestion
from pathlib import Path
```